radixdlt-core/radixdlt/src/main/java/com/radixdlt/hotstuff/epoch/EpochChange.java

```diff
@@ -72,10 +72,12 @@
 public final class EpochChange {
   private final LedgerProof proof;
   private final BFTConfiguration bftConfiguration;
+  private final boolean isShutdown;
 
-  public EpochChange(LedgerProof proof, BFTConfiguration bftConfiguration) {
+  public EpochChange(LedgerProof proof, BFTConfiguration bftConfiguration, boolean isShutdown) {
```

Comment:
I guess it can be a record? ;)

Reply:
it could be :) but I'd rather limit the number of changes on Olympia to minimum :P